### PR TITLE
fix(aio): left align code in aio-code components

### PR DESCRIPTION
**Pull request by petebacondarwin**
**Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: aio/src/app/embedded/code/code.component.ts [hunk 1]
```diff
@@ -118,3 +118,15 @@ export class CodeComponent implements OnChanges {
       (this.code.match(/\n/g) || []).length > 1 : linenums;
   }
 }
+
+function leftAlign(text) {
```
Comment (wardbell) [inline aio/src/app/embedded/code/code.component.ts]: Could be 3 lines shorter but who is counting?
```
function leftAlign(text) {
  const lines = text.split('\n');
  const indent = lines.reduce((minDent, line) => {
    const lineIndent = line.search(/\S/);
    return lineIndent === -1 ? minDent : Math.min(lineIndent, minDent);
  }, Number.MAX_VALUE);
  return lines.map(line => line.substr(indent)).join('\n').trim();
}
```

**Review by wardbell — APPROVED** [review]
Looks great. Just teasing you.

File: aio/src/app/embedded/code/code.component.ts (hunk 1 above)
Comment (gkalpak) [inline aio/src/app/embedded/code/code.component.ts]: Hey, I have a shorter one 😛 : 
```
function leftAlign(text) {
  const lines = text.split('\n');
  const indent = Math.min(Number.MAX_VALUE, ...lines.map(line => line.search(/\S/)).filter(idx => idx !== -1));
  return lines.map(line => line.substr(indent)).join('\n').trim();
}
```

Comment (IgorMinar) [inline aio/src/app/embedded/code/code.component.ts]: shorter doesn't always mean better. always prefer readability and maintainability over shortness

**Review by IgorMinar — APPROVED** [review]

**Pull request merged**